config/_files2.yml

```diff
@@ -0,0 +1,492 @@
+bootstrap:
```

Comment:
I dont think we need this file anymore, do we?

Reply:
No this can be deleted